### PR TITLE
Website release 1.5.1

### PR DESCRIPTION
**Pull request by cjakeman**
Changes to the PHP website to change from v1.4 Release to v1.5.1

Still downloads files from openrails.org/files and not from GitHub.com.

### CODE REVIEW

File: Website/openrails.org/index.php [hunk 1]
```diff
@@ -72,14 +71,14 @@
           </div>
           <div style="background-color: #ffffcc; margin: 0 -5px; padding: 0 5px;">
             <p>
-              <strong>Summer 2022</strong>
-              Three new UK routes are now available for immediate download <a href="http://openrails.org/download/content/">here</a>.
+              <strong>July 2022</strong>
```
Comment (Sharpe49) [inline Website/openrails.org/index.php:74]:
```suggestion
              <strong>November 2022</strong>
```

File: Website/openrails.org/discover/version-1-5-1/index.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        <div class="col-md-12">
+          <h1>Discover > Version 1.5.1</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-10">
+          <h2>New in Open Rails 1.5.1 (since 1.4)</h2>
+          <p>
+            Note: Version 1.5.1 is the same as Version 1.5 but fixes an extra 3 bugs.
```
Comment (twpol) [inline Website/openrails.org/discover/version-1-5-1/index.php:20]: Move this note to discover 1.5 and remove discover 1.5.1 - program links to discover 1.5 page even for 1.5.1

File: Website/openrails.org/shared/menu.php [hunk 3]
```diff
@@ -34,7 +34,8 @@ function in_directory($cwd, $name) {
                 <!--<li><a href="/discover/version-1-1/">Version 1.1</a></li>-->
                 <!-- <li><a href="/discover/version-1-2/">Version 1.2</a></li> -->
                 <!-- <li><a href="/discover/version-1-3/">Version 1.3</a></li> -->
-                <li><a href="/discover/version-1-4/">Version 1.4</a></li>
+                <!-- <li><a href="/discover/version-1-4/">Version 1.4</a></li> -->
+                <li><a href="/discover/version-1-5-1/">Version 1.5.1</a></li>
```
Comment (twpol) [inline Website/openrails.org/shared/menu.php:38]: Put back to discover 1.5 page

**Review by twpol — DISMISSED** [review]
Looks good apart from 1.5.1 -> 1.5 discover page

**Comment by cjakeman** [comment]
> Hi Chris, At line 30 of Website/openrails.org7index.php you have corrected $download_stable = 'OpenRails-1.4-Setup.exe'; to $download_stable = 'OpenRails-1.5-Setup.exe';
> 
> Shouldn't it be 1.5.1 instead of 1.5?

Yes. Thanks, Carlo.

File: Website/openrails.org/index.php (hunk 1 above)
Comment (cjakeman) [reply Website/openrails.org/index.php]: Fixed, thanks.

File: Website/openrails.org/shared/menu.php (hunk 3 above)
Comment (cjakeman) [reply Website/openrails.org/shared/menu.php]: Changes made.

James, please approve again.
Assumubg it's OK, please re-run the script that populates openrails.org, so I can reconcile with Ryan's website repo.

**Review by peternewell — APPROVED** [review]
Hi Chris,

All looks good.

Thanks for your work in maintaining the web site.

Peter

**Pull request merged**